features/fixtures/sampler/scenario_envs/default-anr.env

```diff
@@ -0,0 +1,2 @@
+MAZE_ENABLE_ANR=true
+MAZE_ANR_TIMEOUT=0
```

Comment:
I think this needs a comment to state that values <= 0 won't be set - this was confusing to start with